erase-install.sh

```diff
@@ -1756,7 +1756,7 @@ while test $# -gt 0 ; do
             shift
             prechosen_os="$1"
             ;;
-        --newvolumename) renamevolume="yes"
+        --newvolumename) rename_volume="yes"
```

Comment:
The `rename_volume` key is superfluous. Just set `newvolumename` and then `if [[ $newvolumename ]]`, add the `--newvolumename` option, otherwise don't.

Reply:
Made that change.